pkg/cli/client.go

```diff
@@ -119,11 +122,19 @@ func doHTTPRequest(req *http.Request, verifyTLS bool, resultsPtr interface{}) (h
 
 	httpClientLock.Unlock()
 
+	if debug {
+		fmt.Fprintln(configWriter, req.Method, " ", req.URL)
+	}
+
 	resp, err := httpClient.Do(req)
 	if err != nil {
 		return nil, err
 	}
 
+	if debug {
+		fmt.Fprintln(configWriter, req.Method, req.URL, "Status code ", resp.StatusCode)
```

Comment:
Call it a single word "status".
Print request/response headers also.

Reply:
I have made the changes